logstash-plugins/logstash-input-azure_event_hubs
# Add EventHub user properties in @metadata object

Add EventHub user properties in @metadata object


- enhancement

## Release notes



## What does this PR do?




## Why is it important/What is the impact to the user?

  This PR add EventHub user properties in @metadata object.



## Checklist




## Author's Checklist



## How to test this PR locally


1. Create EventHub message
   * Add body data (any JSON Object)
   * Add user properties as a metadata 
2. Send message to EventHub
3. Consume message from EventHub and check [@metadata][user_properties] 

## Related issues


- 

## Use cases
You can access EventHub metadata in logstash pipeline



## Screenshots



## Logs
